src/troute-network/troute/nhd_preprocess.py

```diff
@@ -14,6 +14,238 @@
 
 LOG = logging.getLogger('')
 
+def read_geo_file(supernetwork_parameters, waterbody_parameters, data_assimilation_parameters):
```

Comment:
Again, if this functionality is in `NHDNetwork` this file should be dropped from the index before merging.

Reply:
Same as above.